xml/MAIN-SBP-scomusermanual.xml

```diff
@@ -2569,7 +2569,7 @@ sles11sp2x64-om2012-sm17:/ #
                         <title>&suma; Documentation</title>
                         <para>Refer to the &suma; documentation <quote>Installable
                                 packages</quote> in the Web interface at <link
-                                    xlink:href="https://www.suse.com/documentation/suse-manager-3/book_suma_reference_manual_3/data/book_suma_reference_manual_3.html"
+                                    xlink:href="https://documentation.suse.com/suma/3.2/"
```

Comment:
Please link to the Reference Guide here.

Reply:
Agree